packages/web-client/ember-cli-build.js

```diff
@@ -3,8 +3,37 @@
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 const path = require('path');
 const concat = require('broccoli-concat');
+const compileCSS = require('broccoli-postcss');
+const postcss = require('postcss');
 const webpack = require('webpack');
 
+const prependSelector = postcss.plugin(
+  'postcss-prepend-selector',
+  function (opts) {
+    opts = opts || {};
+    return function (css) {
+      css.walkRules(function (rule) {
+        rule.selectors = rule.selectors.map(function (selector) {
+          if (/(^|\b)(html|:root)\b/.test(selector)) {
+            return selector;
+          }
+          // eslint-disable-next-line no-useless-escape
+          if (/^([0-9]*[.])?[0-9]+\%$|^from$|^to$/.test(selector)) {
+            // This is part of a keyframe
+            return selector;
+          }
+
+          if (selector.startsWith(opts.selector.trim())) {
+            return selector;
+          }
+
+          return opts.selector + selector;
+        });
+      });
+    };
+  }
+);
+
```

Comment:
I noticed this when running locally:

```
postcss-prepend-selector: postcss.plugin was deprecated. Migration guide:
https://evilmartians.com/chronicles/postcss-8-plugin-migration
```

So I adapted the relevant migration to the Ember CLI context:

```suggestion
const compileCSS = require('broccoli-postcss');
const webpack = require('webpack');

const prependSelector = (opts = {}) => {
  return {
    postcssPlugin: 'postcss-prepend-selector',
    Once(css) {
      css.walkRules(function (rule) {
        rule.selectors = rule.selectors.map(function (selector) {
          if (/(^|\b)(html|:root)\b/.test(selector)) {
            return selector;
          }
          // eslint-disable-next-line no-useless-escape
          if (/^([0-9]*[.])?[0-9]+\%$|^from$|^to$/.test(selector)) {
            // This is part of a keyframe
            return selector;
          }

          if (selector.startsWith(opts.selector.trim())) {
            return selector;
          }

          return opts.selector + selector;
        });
      });
    },
  };
};

```

It seems to have worked!

![image](https://user-images.githubusercontent.com/43280/142911451-20d65367-8805-44c0-a418-cacde75d1834.png)

Reply:
the [migration guide](https://evilmartians.com/chronicles/postcss-8-plugin-migration) has more after that actually, including changing `Once` to `AtRule`, might be worth looking more deeply into. I’m surprised `Once` didn’t need to be imported…???